Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement.py

```diff
@@ -436,6 +452,23 @@ def reset_auth(client: Client) -> CommandResults:
                                           '**!azure-sql-auth-start** and **!azure-sql-auth-complete**.')
 
 
+@logger
+def test_module(client, _):
+    """
+    Performs basic GET request to check if the API is reachable and authentication is successful.
+    Returns ok if successful.
+    """
+    if demisto.params().get('auth_type') == 'Device':
```

Comment:
same question here about running the test command

Reply:
same answer, we must set the instance first.